index.html

```diff
@@ -0,0 +1,416 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Smashing Magazine</title>
+    <link href="https://fonts.googleapis.com/css?family=Dosis&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+
+  <body>
+
+    <header>
+      <div class="logo"><h3>SMASHING</h3>MAGAZINE</div>
+     
+      <div class="nav">
```

Comment:
[OPTIONAL]

- [ ] I strongly recommend changing this `<div>` tag to a `<nav>` tag because is more Semantic HTML coherent

Reply:
implemented optional recommendations